i18n/testdata/translation_mismatches.noargs.es.po

```diff
@@ -0,0 +1,41 @@
+# Testing
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2020-03-25 13:57+0000\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+
+#: Translated/43f7e69e-727d-4cfe-81b8-564e7833052b/name:0
+#: Translated/e42deebf-90fa-4636-81cb-d247a3d3ba75/quick_replies:1
```

Comment:
I like this for readability but you don't want the UUID in there still? Guess we can say don't rename your flows after generating PO files though.

Reply:
To be clear.. we're not going to use this programmatically for anything.. it's just a hint to a translator... and I'm leaning to toward the simplicity of not having two UUIDs in there. I'm thinking we'd use `msgctxt` to match a specific node to update with a new translation..

Reply:
Ok, ya, that makes sense and is consistent with the use of those comments in other tools. 👍

Reply:
We can experiment in future - maybe Transifex would work with URLs, and the editor could accept URLs with `#` links to specific parts of flows 🤔